README.md

```
@@ -0,0 +1,89 @@
+# **Bank Application**
+
+Deployment of a Banking Application using AWS Services
+
+> ![Bank App 1](/assets/bank-app-1.png)
+>
+>![Bank App 2](/assets/bank-app-2.png)
+
+### **Application Structure**
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)